src/effects/native/loudnesscontoureffect.cpp

```diff
@@ -0,0 +1,159 @@
+#include <effects/native/loudnesscontoureffect.h>
+#include "util/math.h"
+
+namespace {
+
+// The defaults are tweaked to match the TEA6320 IC
+static const int kStartupSamplerate = 44100;
+static const double kLoPeakFreq = 20.0;
+static const double kHiShelveFreq = 3000.0;
+static const double kMaxLoGain = 30.0;
+static const double kHiShelveGainDiv = 3.0;
+static const double kLoPleakQ = 0.2;
+static const double kHiShelveQ = 0.7;
+
+} // anonymous namesspace
+
+
+// static
+QString LoudnessContourEffect::getId() {
+    return "org.mixxx.effects.loudnesscontour";
+}
+
+// static
+EffectManifest LoudnessContourEffect::getManifest() {
+    EffectManifest manifest;
+    manifest.setId(getId());
+    manifest.setName(QObject::tr("Loudness Contour"));
+    manifest.setShortName(QObject::tr("Loudness"));
+    manifest.setAuthor("The Mixxx Team");
+    manifest.setVersion("1.0");
+    manifest.setDescription(QObject::tr(
+        "A filter to compensate the loudness contour at lower volume"));
+    manifest.setEffectRampsFromDry(true);
+    manifest.setIsMixingEQ(true);
+
+    EffectManifestParameter* low = manifest.addParameter();
```

Comment:
Since you added gain tracking, do we need this parameter? What's the use case if the master-gain-tracking perfectly adjusts the curve to compensate? Usually stereos just have a "loudness" button and not a manual-adjust knob, right?